src/jsonformat.app.src

```diff
@@ -15,7 +15,7 @@
 {application, jsonformat,
   [ {description,  "A custom formatter for the erlang logger applications that "
                    "outputs JSON formatted single line logs"}
-  , {vsn,           "1.0.1"}
+  , {vsn,           "0.0.1"}
```

Comment:
just resetting to default since we don't usually change this

Reply:
Don't we normally put `vsn` to the atom `git`?

Reply:
good point!